compatibility_server/pip_checker.py

```diff
@@ -38,6 +38,9 @@
 PYPI_URL = 'https://pypi.org/pypi/'
 CONTAINER_WITH_PKG = "checker"
 TIME_OUT = 300  # seconds
+GITHUB_CLIENTLIBS_PREFIX = 'git+git://github.com/googleapis/' \
```

Comment:
Is this really called client libs? I thought that was https://github.com/googleapis/google-api-python-client

Reply:
It's https://github.com/googleapis/google-cloud-python actually, I just call it clientlibs for short. That one is the discovery based APIs.